routers/api/v1/repo/pull.go

```diff
@@ -15,6 +15,7 @@ import (
 	"code.gitea.io/gitea/modules/log"
 
 	api "code.gitea.io/sdk/gitea"
+	"net/http"
```

Comment:
import order https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
^ this (move `net/http` up!)